ansible/roles/operator-pipeline/templates/openshift/tasks/set-github-status.yml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: set-github-status
+spec:
+  params:
+    - name: git_repo_url
+    - name: commit_sha
+      description: SHA of the commit to set the status for.
+    - name: description
+      description: Description attached to the github status.
+    - name: state
+      default: ""
+      description: The github status to set for the given commit. Only used if
+        test_result not provided.
+    - name: context
+      description: Context attached
+    - name: preflight_results_exists
+      default: "false"
+    - name: test_result
+      default: ""
+      description: Path to the file that contains test results.
+    - name: result_url
+      default: ""
+      description: URL to the test result on Pyxis.
+  workspaces:
+    - name: github-bot-token
+  steps:
+    - name: set-github-status
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
+      script: |
+        #! /usr/bin/env bash
+        # Don't use set -x to avoid exposing github token
+        set -e
+
+        PREFLIGHT_RESULTS_EXISTS="$(params.preflight_results_exists)"
```

Comment:
nitpick: you could use `steps.environment` section like
```
steps:
   - name:  set-github-status
     environment:
       - name: PREFLIGHT_RESULTS_EXISTS
         value: $(params.preflight_results_exists)
```

Reply:
I would also change the `PREFLIGHT_RESULTS_EXISTS` to `SKIP`, to keep the task generic :)

Reply:
This is actually not needed anymore, since we can just use the new `params.skip` directly